core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -92,6 +92,7 @@ class ContainerProxy(
     factory: (TransactionId, String, ImageName, Boolean, ByteSize) => Future[Container],
     sendActiveAck: (TransactionId, WhiskActivation, InstanceId) => Future[Any],
     storeActivation: (TransactionId, WhiskActivation) => Future[Any],
+    instance: Int,
```

Comment:
When you make that change, remove all the `toString` occurences you created and simply go for 

`s"wsk${instance.toInt}..."`